stoplight.js

```diff
@@ -0,0 +1,13 @@
+// console.log("stoplight says hello");
+$(document).ready(function(){
+  $("#red").on("click", function(){
+    $(this).toggleClass("active");
+  })
+  $("#yellow").on("click", function(){
+    $(this).toggleClass("active");
+  })
+  $("#green").on("click", function(){
+    $(this).toggleClass("active");
+  })
+
```

Comment:
how would we make this just one event? is there anything that all three of these share?